style.css

```diff
@@ -0,0 +1,69 @@
+    header {
+        position: fixed;
+        background-color: #ffffff;
+        width: 100%;
+        height: 70px;
+    }
+    
+    body {
+        background-color: black;
+        margin: 0px;
+    }
+    
+    .logo {
+        width: 70px;
+        height: 11px;
+        margin-top: 27px;
+        margin-left: 240px;
+    }
+    
+    .menu {
+        font-family: 'Lato', sans-serif;
+        font-weight: 700;
+        font-size: 11px;
+        text-transform: uppercase;
+        margin-top: -27px;
+        margin-left: 361px;
+    }
+    
+    .tag-menu {
+        text-decoration: none;
+        color: #797979;
+    }
+    
+    .menu>p {
+        float: left;
```

Comment:
Why do you need float here? It's wrong. This property now isn't being used, almost.